extensions/azurecore/src/account-provider/auths/azureAuth.ts

```diff
@@ -119,7 +119,8 @@ export abstract class AzureAuth implements vscode.Disposable {
 				const token: Token = {
 					token: result.response.accessToken,
 					key: result.response.account.homeAccountId,
-					tokenType: result.response.tokenType
+					tokenType: result.response.tokenType,
+					expiresOn: result.response.expiresOn!.getTime() / 1000
```

Comment:
Be careful with using non-null assertions here. If the response is actually guaranteed to have a value for expiresOn (and specifically a date value) then it's likely the typings should be fixed there too.